scripts/oneoff/remove-dos4-questions-from-dos5-drafts.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+
+sys.path.insert(0, '.')
+from dmapiclient import DataAPIClient
+from dmscripts.helpers.auth_helpers import get_auth_token
+from dmutils.env_helpers import get_api_endpoint_from_stage
+
+from typing import List
+
+
+def get_affected_drafts_services(api_client: DataAPIClient) -> List[dict]:
+    all_drafts = api_client.find_draft_services_by_framework(
+        'digital-outcomes-and-specialists-5', status='not-submitted'
+    )["services"]
+
+    return [draft for draft in all_drafts if draft_service_contains_dos4_answer(draft)]
+
+
+def draft_service_contains_dos4_answer(draft: dict) -> bool:
+    invalid_answers = {
+        "agileCoachPriceMin",
+        "businessAnalystPriceMin",
+        "communicationsManagerPriceMin",
+        "contentDesignerPriceMin",
+        "dataArchitectPriceMin",
+        "dataEngineerPriceMin",
+        "dataScientistPriceMin",
+        "deliveryManagerPriceMin",
+        "designerPriceMin",
+        "developerPriceMin",
+        "performanceAnalystPriceMin",
+        "portfolioManagerPriceMin",
+        "productManagerPriceMin",
+        "programmeManagerPriceMin",
+        "qualityAssurancePriceMin",
+        "securityConsultantPriceMin",
+        "serviceManagerPriceMin",
+        "technicalArchitectPriceMin",
+        "userResearcherPriceMin",
+        "webOperationsPriceMin",
+        "developerAccessibleApplications",
+        "contentDesignerAccessibleApplications",
+        "qualityAssuranceAccessibleApplications",
+        "technicalArchitectAccessibleApplications",
+        "webOperationsAccessibleApplications",
+        "serviceManagerAccessibleApplications",
+        "designerAccessibleApplications"
+    }
+
+    return invalid_answers.intersection(draft)
```

Comment:
We should coerce this to `bool` to make the typing valid

Reply:
```suggestion
    return bool(invalid_answers.intersection(draft))
```